benchmarks/bvh_driver/CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
-set(POINT_CLOUDS_INCLUDE_DIR ${CMAKE_SOURCE_DIR}/benchmarks/point_clouds)
+set(BENCHMARKS_UTILS_INCLUDE_DIR ${CMAKE_SOURCE_DIR}/benchmarks/utils)
```

Comment:
```suggestion
set(ARBORX_BENCHMARKS_UTILS_INCLUDE_DIR ${CMAKE_SOURCE_DIR}/benchmarks/utils)
```
to be on the safe side

Reply:
Actually should we say BENCHMARK[S] without the s?

Reply:
> Actually should we say BENCHMARK[S] without the s?

No strong opinion.